sg_DopplerSpatAlgorithm.hpp

```diff
@@ -19,7 +19,7 @@
 
 #pragma once
 
-#include "AlgoGRIS/Data/sg_SpatMode.hpp"
+#include "Data/sg_SpatMode.hpp"
```

Comment:
Should this be 
```suggestion
#include <Data/sg_SpatMode.hpp>
```

instead ? The rest of the includes are in that format ...

Reply:
ah yep totally